src/ModelParser/RawMetadata/PropertyVariationMetadata.php

```diff
@@ -87,6 +87,17 @@ public function setVersionRange(VersionRange $version): void
         parent::setVersionRange($version);
     }
 
+    /**
+     * The value can be anything that the consumer understands.
+     *
+     * However, if it is an object, it should implement JsonSerializable to not
+     * break debugging.
```

Comment:
Do we need some "validation" of `$value`?

Reply:
i will add a check in json serialization for it.

Reply:
handling that case in AbstractPropertyMetadata::jsonSerialize now